src/tests/functional/shared_test_classes/src/single_layer/shape_of.cpp

```diff
@@ -25,7 +25,7 @@ namespace LayerTestsDefinitions {
         auto inType = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(inputPrecision);
         auto param = ngraph::builder::makeParams(inType, {inputShapes});
         auto paramOuts = ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::opset3::Parameter>(param));
-        auto shapeOf = std::make_shared<ngraph::opset3::ShapeOf>(paramOuts[0], inType);
+        auto shapeOf = std::make_shared<ngraph::opset3::ShapeOf>(paramOuts[0]);
```

Comment:
you should add additional parameter, not to remove `inType`. Please, check the bug description again: `std::tie(inputPrecision, outType, inputShapes, targetDevice) = this->GetParam();`
...
`auto shapeOf = std::make_shared<ngraph::opset3::ShapeOf>(paramOuts[0]);`

Reply:
The constructor of the ngraph::opset3::ShapeOf class has the output_type argument, which is equal to element::i64 by default. 

https://github.com/openvinotoolkit/openvino/blob/2f9c5df2717db5f37768c3230749ace79236dc4b/src/core/include/openvino/op/shape_of.hpp#L19

In my opinion this is correct logic and there is no need to pass a custom value in the SetUp() method in the test framework.

Reply:
The constructor of the ngraph::opset3::ShapeOf class has the output_type argument, which is equal to element::i64 by default. 

https://github.com/openvinotoolkit/openvino/blob/2f9c5df2717db5f37768c3230749ace79236dc4b/src/core/include/openvino/op/shape_of.hpp#L19

In my opinion this is correct logic and there is no need to pass a custom value in the SetUp() method in the test framework.